pysparkle/pysparkle/data_quality/data_quality.py

```diff
@@ -0,0 +1,35 @@
+import logging
```

Comment:
I think __init__.py is missing for data_quality package - the app is failing while importing

Reply:
Yup writing the unit tests now so caught this and a couple of other things. The pipeline run issue won't be fixed till this afternoon so I'll just put the unit tests and fixes in this PR aswell. I'll ping you when it's ready for another look @motylp